src/Microsoft.Health.Fhir.Core/Features/Validation/NarrativeValidator.cs

```diff
@@ -25,11 +26,25 @@ public NarrativeValidator(INarrativeHtmlSanitizer narrativeHtmlSanitizer)
             _narrativeHtmlSanitizer = narrativeHtmlSanitizer;
         }
 
-        public override IEnumerable<ValidationFailure> Validate(PropertyValidatorContext context)
+        public override string Name => "NarrativeValidator";
```

Comment:
>public override string Name => "NarrativeValidator"; [](http://example.com/codeflow?start=8&length=52)

How it's used? And is it needed?

Reply:
It is an abstract member of PropertyValidator class which we need to implement here
I think it is used in case of any errors while building an error message

Reply:
Ok.

I would go with `nameof(NarrativeValidator<T, TProperty>)` instead constant string.

Reply:
Good idea! Updated